spec/rubocop/cop/style/safe_navigation_spec.rb

```diff
@@ -44,16 +44,41 @@
       expect_no_offenses('foo && !foo.bar.baz?')
     end
 
-    it 'allows method call that are used in a comparison safe guarded by ' \
+    it 'allows method call that is used in a comparison safe guarded by ' \
       'an object check' do
       expect_no_offenses('foo.bar > 2 if foo')
     end
 
-    it 'allows an object check before a method call that are used in ' \
+    it 'allows method call that is used in a non standard comparison ' \
+      'safe guarded by an object check' do
+      expect_no_offenses('foo.bar !~ /baz/ if foo')
+    end
+
+    it 'allows an object check before a method call that is used in ' \
       'a comparison' do
       expect_no_offenses('foo && foo.bar > 2')
     end
 
+    it 'allows an object check before a method call that is used in ' \
+      'a non standard comparison' do
+      expect_no_offenses('foo && foo.bar !~ /baz/')
+    end
+
+    it 'allows an object check before a method chain that is used in ' \
+      'a comparison' do
+      expect_no_offenses('foo && foo.bar > 2')
+    end
```

Comment:
It seems to be the same test as L57.
https://travis-ci.org/bbatsov/rubocop/jobs/326143335#L1090-L1097